src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -29,7 +29,21 @@
   {{ form_start(currencyForm, {
     'attr': {
       'id': 'currency_form',
-      'data-get-cldr-data': path('admin_currencies_get_reference_data', {'currencyIsoCode': 'CURRENCY_ISO_CODE'}),
+      'data-reference-url': path('admin_currencies_get_reference_data', {'currencyIsoCode': 'CURRENCY_ISO_CODE'})|replace({'/CURRENCY_ISO_CODE': '{/id}'}),
+      'data-languages': languages|json_encode|replace("'", "&#39;")|raw,
+      'data-translations': {
+        '1. Enter symbol': '1. Enter symbol'|trans({}, 'Admin.International.Feature'),
```

Comment:
Because of vue js, you can do something more easier
```suggestion
        'form.symbol': '1. Enter symbol'|trans({}, 'Admin.International.Feature'),
```
Use simple keys

Reply:
The difference is in PS we use the original english string, which is not the same in Vue apps :)